### PR TITLE
Common UI

### PR DESCRIPTION
**Pull request by lindlands**
Base common UI. Works well on web and mobile! whoa

### CODE REVIEW

**Comment by dylantompkins** [comment]
Running `npm ci` and then `npm run web` fails to build. Please resolve.

20240317_23h30m53s_grim

**Comment by dylantompkins** [comment]
Running `npm ci` and then `npm run web` fails to build. Please resolve and then request my review again.

20240317_23h30m53s_grim

**Comment by dylantompkins** [comment]
Working now, but I noticed that this blob is going over the text box. Can you please put it under? Thanks

20240317_23h38m35s_grim

**Comment by dylantompkins** [comment]
image
image
image
image

**Comment by dylantompkins** [comment]
1. This is the farthest I can scroll down on the facility profile page. It's awkward to be able to click the button at the bottom. Please add padding
2. A warning I get when visiting the ReviewForm
3. Another ReviewForm warning
4. UserProfile looks weird

**Comment by dylantompkins** [comment]
also you are missing a dependency. please run `npx yarn add react-native-gesture-handler`

**Comment by dylantompkins** [comment]
Looks really good but I can't see the facility profile or reviews page because my map PR removed our ability to navigate to it. Oops! That's my bad. I'm going to make a quick PR to add a single hardcoded icon on the map so that we can get to those pages again. I will request your review on it.

**Comment by dylantompkins** [comment]
please approve https://github.com/Green-Kittens/bathroom-finder-frontend/pull/74 so that I can see the whole effect of your pr

**Comment by dylantompkins** [comment]
```
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
```

File: navigation/screens/facility-profile.tsx
[the remark this answers is not in the document]
Comment (lindlands) [reply navigation/screens/facility-profile.tsx:1]: Plan is to use rneui component instead, images are placeholder for now. Follow up [here](https://github.com/Green-Kittens/bathroom-finder-frontend/issues/75).

[the remark this answers is not in the document]
Comment (lindlands) [reply navigation/screens/facility-profile.tsx:1]: Decided against figuring it out for this PR as to not to increase scope of PR. Follow-up task [here](https://github.com/Green-Kittens/bathroom-finder-frontend/issues/76).

**Review by dylantompkins — APPROVED** [review]

**Pull request merged**